test/widgets/action_sheet_test.dart

```diff
@@ -99,6 +103,302 @@ void main() {
     connection.prepare(httpStatus: 400, json: fakeResponseJson);
   }
 
+  group('showTopicActionSheet', () {
+    final channel = eg.stream();
+    const topic = 'my topic';
+    final message = eg.streamMessage(
+      stream: channel, topic: topic, sender: eg.otherUser);
+
+    Future<void> prepare() async {
+      addTearDown(testBinding.reset);
+
+      await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot(
+        realmUsers: [eg.selfUser, eg.otherUser],
+        streams: [channel],
+        subscriptions: [eg.subscription(channel)]));
+      store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
+      connection = store.connection as FakeApiConnection;
+
+      await store.addMessage(message);
+    }
+
+    testWidgets('show from inbox', (tester) async {
+      await prepare();
+      await tester.pumpWidget(TestZulipApp(accountId: eg.selfAccount.id,
+        child: const InboxPage()));
+      await tester.pump();
+
+      await tester.longPress(find.text(topic));
+      // sheet appears onscreen; default duration of bottom-sheet enter animation
+      await tester.pump(const Duration(milliseconds: 250));
+      check(find.byType(BottomSheet)).findsOne();
+    });
```

Comment:
We can be a bit more specific in pinning down that the bottom sheet that comes up is the right bottom sheet:
```suggestion
      check(find.byType(BottomSheet)).findsOne();
      check(find.text('Follow topic')).findsOne();
    });
```